kolibri/core/assets/src/views/CoreBanner.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+
+  <div>
+    <div
+      class="banner"
+      :style="{ background: $coreBgLight}"
+      :tabindex="0"
+      role="dialog"
+    >
+      <div class="banner-inner">
+        <KGrid>
+          <!-- Grid Content -->
+          <KGridItem :sizes="contentGridSizes">
+            <slot :bannerClosed="bannerClosed"></slot>
+          </KGridItem>
+
+          <!-- Grid Buttons -->
+          <KGridItem v-if="!bannerClosed" sizes="4, 8, 12">
+            <KButton
+              v-if="!bannerClosed"
+              class="close-button"
+              :text="$tr('closeButton')"
+              appearance="flat-button"
+              :primary="true"
+              @click="toggleBanner"
+            />
+          </KGridItem>
+          <KGridItem
+            v-else
+            sizes="4, 2, 2"
+            alignment="right"
+          >
+            <KButton
+              class="open-button"
+              :text="$tr('openButton')"
+              appearance="flat-button"
+              :primary="true"
+              @click="toggleBanner"
+            />
+          </KGridItem>
+        </KGrid>
+      </div>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import themeMixin from 'kolibri.coreVue.mixins.themeMixin';
+  import KButton from 'kolibri.coreVue.components.KButton';
+  import KGrid from 'kolibri.coreVue.components.KGrid';
+  import KGridItem from 'kolibri.coreVue.components.KGridItem';
+
+  export default {
+    name: 'CoreBanner',
+    components: { KButton, KGrid, KGridItem },
+    mixins: [themeMixin],
+    data() {
+      return {
+        bannerClosed: false,
+      };
+    },
+    computed: {
+      contentGridSizes() {
+        return this.bannerClosed ? '4, 6, 10' : '4, 8, 12';
+      },
+    },
+    methods: {
+      toggleBanner() {
+        this.bannerClosed = !this.bannerClosed;
+      },
+    },
+    $trs: {
+      openButton: 'More Info',
+      closeButton: 'Close',
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .banner {
+    position: relative;
+    top: 0%;
+    left: 0%;
+    z-index: 7500;
+    width: 100%;
+    margin: 0 auto;
+    overflow-y: auto;
+    box-shadow: 0 2px 50px rgba(0, 0, 0, 1);
```

Comment:
we have a set of Material-base built-in drop-shadows. In this case, we should match the login pane lower on the page:

https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/user/assets/src/views/SignInPage/index.vue#L477


I think I think `top`, `left`, `overflow-y`, and `z-index` are no longer necessary?